python/paddle/fluid/contrib/mixed_precision/fp16_utils.py

```diff
@@ -35,6 +35,30 @@ def _rename_arg(op, old_name, new_name):
     op_desc._rename_output(old_name, new_name)
 
 
+def _change_optimize_ops_order(block):
```

Comment:
_move_optimize_ops_back(block) may be better as a more clear name

Reply:
Done. Thanks!